Applications/Spire/Source/Ui/ListItem.cpp

```diff
@@ -49,6 +50,9 @@ bool ListItem::is_selected() const {
 }
 
 void ListItem::set_selected(bool is_selected) {
+  if(!isEnabled()) {
```

Comment:
This violates both the API and the model. ```ListItem``` is a view class, it has absolutely no authority to override the model about whether the value it represents is selected or not.